README.md

```diff
@@ -106,6 +106,10 @@ class MyComp extends Vue {
 }
 ```
 
+#### Process Initial Property on Custom Decorators
+
+If you pass the object that has `getInitialProperty: true` option, `createDecorator`'s callback will receive initial value of each property as the 3rd argument. Note that the properties will no longer be collected as component data in that case.
```

Comment:
> If you pass an object with `getInitialProperty: true` field, 

A grammatical change.

IMHO placing an example or a link to test case is more clear than a piece of description.